src/components/common/Tabs/TabItem.tsx

```diff
@@ -3,20 +3,13 @@ import styled from 'styled-components'
 
 interface TabProps {
   title: string
-  readonly id: number
-  onTabClick: (arg: number) => void
-  isActive: boolean
-  readonly activeColor?: string
+  onTabClick: () => void
+  readonly activeColor: string | false
```

Comment:
I see that writing 
`readonly activeColor: string | false`
is more explicit than doing:
`readonly activeColor?: string`

Because in the latter, if you don't pass anything, we wouldn't know the type?

Now if no `activeColor` value is passed, it falls back to being `false`.

Reply:
i didn't see a scenario where we would leave out the `activeColor` - and the pattern `someTruthyValue && valueIfTruthy` is common enough. To make it agree with TS we need to type it this way. You could also always do `someTruthyValue ? valueIfTruthy : ' '` if that's more clear. Then you could leave the type as `string | undefined` - but the base type where activeColor came from (i think Tabs?) had `activeColor` set as `string | undefined` and i saw no such scenario where `activeColor` would ever be undefined

Reply:
Thinking about it as a 'generic' Tabs component, the only scenario I can think of for not wanting to pass the activeColor, is if you want to use the default colors. Which...seems to me to be a quite common scenario?

E.g. in this case we use Tabs to have the Green/Red (buy/sell) appearance. But say, if you have a 'normal' Tab layout, you'd want to just use the default colors (not Green/Red).

Reply:
I think the best approach depends on how you want to re-use the components. One way would be to use theme colours with alternatives. Then use an `isActive` prop that toggles the highlight colour internally OR just pass an `activeColour` prop that sets it automically, overwriting the original theme colour. But we didn't need both in this case

Reply:
To @Velenir 's point, by making it optional, you always rely on the default/fallback theming for that tabs. Would that be a better approach?

I guess @W3stside you were suggesting to just use a default theme color/variable where the logic is handled elsewhere? You feel that's easier to support optional passed activeColor vars?